src/views/CreateResumeInputInfo.vue

```diff
@@ -674,10 +623,114 @@ async function addNewSkill() {
 }
 
 function filterPerfectMatch(value, search) {
-    console.log("value: " + value + ", search: " + search);
+    // console.log("value: " + value + ", search: " + search);
     return value != null && String(value) === search
     }
 
+async function addResume() {
+    var linkArr = [];
+    var goalArr = [];
+    var eduArr = [];
+    var expArr = [];
+    var skillArr = [];
+
+    if (selectedLinks.value !== null) {
+        for (let [key, value] of Object.entries(selectedLinks.value)) {
+            // console.log("Link Key: " + key + " Value: " + value);
+            linkArr.push(value);
+        }
+        console.log(linkArr);
```

Comment:
some of the arrays/lists are logged, some are not, seems haphazard

also, if your going to log the array/list value for debugging purposes it would probably help to log more contextual information, e.g.,
`console.log(``Before adding resume collected links as ${linkArr}``);`